pages/linux/pivpn.md

```diff
@@ -0,0 +1,29 @@
+# pivpn
+
+> Easy security-hardened OpenVPN setup and manager.
+> Originally designed for the Raspberry Pi, but works on other Linux devices too.
+> Homepage: <http://www.pivpn.io/>.
+
+- Install PiVPN:
```

Comment:
We usually do not mention install steps. Let us remove this.

Reply:
Can do!